video/out/wayland_common.c

```diff
@@ -37,6 +39,23 @@
 // Generated from xdg-decoration-unstable-v1.xml
 #include "video/out/wayland/xdg-decoration-v1.h"
 
+// Generated from presentation-time.xml
+#include "video/out/wayland/presentation-time.h"
+
+#define OPT_BASE_STRUCT struct wayland_opts
+const struct m_sub_options wayland_conf = {
+    .opts = (const struct m_option[]) {
+        OPT_INTRANGE("wayland-frame-offset", frame_offset, 0, -500, 3000),
```

Comment:
Should this not be wayland-frame-wait-offset (going by the documentation)

Reply:
It should be. I apparently messed up. Thanks for catching it!